binaries/llm-cli/src/cli_args.rs

```diff
@@ -332,6 +332,14 @@ pub struct ModelLoad {
     #[arg(long, short = 'm')]
     pub model_path: PathBuf,
 
+    /// Where to load the vocabulary from
+    #[arg(long, short = 'v')]
+    pub vocabulary_path: Option<PathBuf>,
+
+    /// Where to load the vocabulary from
+    #[arg(long, short = 'r')]
+    pub vocabulary_repo: Option<String>,
+
```

Comment:
I haven't looked yet so it's possible this is well defined, but it should be clearly stated in the docs which one of these will take precedence if they are both present.